Automattic/jetpack
# Jetpack:  Fix visual issues in the Product Price component

This PR adds literal CSS classes to the ProductPrice component of the RNA and uses them to apply styles in the Jetpack plugin context in the traditional way (not using CSS modules).

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack:  Fix visual issues in the Product Price component
  * Add literal CSS class to the ProductPrice component
  * Consolidate current styles by using literal CSS class in Jetpack plugin

#### Other information:

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Jetpack plugin, some of the product pages
  * http://localhost/wp-admin/admin.php?page=jetpack#/product/scan
  * http://localhost/wp-admin/admin.php?page=jetpack#/product/videopress
  * etc
* Confirm the styles are look as expected now.

before | after
------|------
<img width="812" alt="image" src="https://user-images.githubusercontent.com/77539/197584868-b7e64d06-e8c6-47c4-99ca-6e2993ea7964.png">| <img width="819" alt="image" src="https://user-images.githubusercontent.com/77539/197584596-f4fad428-8d5f-41b5-bc13-bbe8f82e3b56.png">
